generators/generator-constants.cjs

```diff
@@ -33,7 +33,7 @@ const javaDependencies = {};
 Object.freeze(javaDependencies);
 
 // The version should be coherent with the one from spring-data-elasticsearch project
-const ELATICSEARCH_TAG = '7.17.4';
+const ELATICSEARCH_TAG = '8.5.0';
```

Comment:
Healthy check is not working at new version 
```suggestion
const ELATICSEARCH_TAG = '7.17.4';
```

Reply:
@mshima spring-boot 3 target that version

Reply:
I understand.
CI is passing with v7.
So we should create a separated PR to update elasticsearch and fix the health check.

Reply:
Opened https://github.com/jhipster/generator-jhipster/pull/20463 to track